Makefile

```diff
@@ -37,3 +37,7 @@ vet:
 
 test-output:
 	$(shell echo $$GO_VERSION_NUMBER)
+
+.PHONY: fmt-fix
+fmt-fix:
+	gofmt -s -w -l .
```

Comment:
If we want to vendor (I had to do this for some stuff recently even in 1.16) this will hit us, but that might be a thing to fix then.